num2words/lang_BG.py

```diff
@@ -0,0 +1,342 @@
+    'шест': 'шести',
+    'седем': 'седми',
+    'осем': 'осми',
+    'девет': 'девети',
+    'десет': 'десети',
+}
+
+
+class Num2Word_BG(Num2Word_Base):
+
+    negword = 'минус'
+
+    def __init__(self):
+        self.is_currency = False
+
+    def set_high_numwords(self, *args):
+        pass
+
+    def merge(self, curr, next):
+        pass
+
+    def pluralize(self, number, forms):
+        is_feminine = forms[-1]
+
+        if number == 1:
+            return forms[0]
+
+        if is_feminine:
+            form = 2
+        else:
+            if number % 10 == 1 and not self.is_currency:
+                form = 0
+            else:
+                form = 1
+
+        return forms[form]
+
+    def setup(self):
+        pass
+
+    def _int2word(self, number, feminine=False):
+        if number < 0:
+            return ' '.join([MINUS_PREFIX_WORD, self._int2word(abs(number))])
+
+        if number == 0:
+            return ZERO
+
+        words = []
+        self.chunks = list(splitbyx(str(number), 3))
+        chunk_len = len(self.chunks)
+
+        for chunk in self.chunks:
+            chunk_len -= 1
+            digit_right, digit_mid, digit_left = get_digits(chunk)
+
+            if digit_left > 0:
+                # the True is missing condition
+                if len(words) and True and digit_right == 0 and digit_mid == 0:
+                    words.append(CONJUNCTION)
+
+                words.append(HUNDREDS[digit_left][0])
+
+            if digit_mid > 1:
+                if len(words) and digit_right == 0:
+                    words.append(CONJUNCTION)
+
+                words.append(TWENTIES[digit_mid][0])
+
+            if digit_mid == 1:
+                if len(words) and digit_right == 0:
+                    pass
+                    words.append(CONJUNCTION)
+
+                words.append(TENS[digit_right][0])
+            elif digit_right > 0:
+                if len(words):
+                    words.append(CONJUNCTION)
+
+                # digit gender depends from scale
+                gender_type = 0
+
+                is_feminine = feminine or SCALE[chunk_len][-1]
+
+                if is_feminine:
+                    gender_type = 2
+                elif (len(self.chunks) > 2 and chunk_len > 1) or self.is_currency:
+                    gender_type = 1
+
+                if chunk_len == 1 and len(self.chunks) == 2:
+                    if digit_mid == 0 and digit_left == 0 and digit_right == 1:
+                        pass
+                    else:
+                        words.append(
+                            ONES[digit_right][gender_type]
+                        )
+                else:
+                    words.append(
+                        ONES[digit_right][gender_type]
+                    )
+
+            if chunk_len > 0 and chunk != 0:
+                words.append(self.pluralize(chunk, SCALE[chunk_len]))
+
+        return ' '.join(words)
+
+    def to_cardinal(self, number, feminine=False):
```

Comment:
It is not a good decision to put boolean feminine parameter, because you have 3 genders - masculine, feminine and neuter. Later it will be a problem with backward compatibility.
Like [here](https://github.com/savoirfairelinux/num2words/commit/0cb3a44dd2362f93c3e45893442e8e67cb7c86d1).
I suggest adding the gender parameter.